onnx/onnx.proto

```diff
@@ -85,45 +85,55 @@ message NodeProto {
 // This is the equivalent of the "network" and "graph" in many deep learning
 // frameworks.
 message GraphProto {
-  // The nodes in the graph.
-  repeated NodeProto node = 1;
-
-  // The name of the graph.
-  optional string name = 2;   // namespace Graph
-
-  // The inputs and outputs of the graph.
-  repeated string input = 3;  // namespace Tensor
-  repeated string output = 4; // namespace Tensor
-
-  // An optional list of initial values for tensors in the list of input above.
-  // Used to pass serialized parameters for networks.
-  // Each entry specifies a value for the input whose TensorProto.name matches
-  // a name in the input. This list is not required to be the same length of as
-  // input since many tensors might not have initial values.
-  repeated TensorProto initializer = 5;
-
   // The version of the IR this graph targets. See Version enum below.
-  optional int64 ir_version = 6;
-
-  // The version of the framework runtime that generates this graph.
-  // This producer_version has the same format as ir_version. 
-  optional int64 producer_version = 7;
-
+  // This field MUST be present. 
+  optional int64 ir_version = 1;
+  
   // The name of the framework used to generate this graph in the form
   // "framework_name[-tag]". Tag is optional and provides additional
   // information such as `alpha` or `beta` or `rc3`.
```

Comment:
This comment is out of date; now that we have a free form string field, we should just put the tag in `producer_version`. So, the `producer_tag` should just be something like `pytorch` or `cntk`.

With this in mind, it might be better to rename this to just `producer` (rather than `producer_tag`). I also wonder if we shouldn't also fold in the version number with producer, so that we have one less field here (I'm ambivalent about this though.)

Reply:
Agreed on the rename and changing the comment text to move any versiony stuff to producer_version. I thought about doing this earlier but I was trying to minimize the BC.  As a nit, I'd use "producer_name" rather than "producer" but I'm not super convicted. 

Ambivalent about combining the fields. 

Unless someone objects, I'll push the producer_tag->producer_name rename.

Reply:
I just pushed the change suggested above.